networktest.go

```diff
@@ -489,18 +493,18 @@ func (n *networkHarness) WaitForTxBroadcast(txid wire.ShaHash) {
 }
 
 // OpenChannel attemps to open a channel between srcNode and destNode with the
-// passed channel funding paramters.
+// passed channel funding parameters.
 func (n *networkHarness) OpenChannel(ctx context.Context,
 	srcNode, destNode *lightningNode, amt btcutil.Amount,
 	numConfs uint32) (lnrpc.Lightning_OpenChannelClient, error) {
 
 	// TODO(roasbeef): should pass actual id instead, will fail if more
 	// connections added for Alice.
 	openReq := &lnrpc.OpenChannelRequest{
-		TargetPeerId:        1,
+		TargetPeerId:        int32(destNode.nodeId),
```

Comment:
This won't necessarily always work as peer's "relative" ID may change with disconnects/connects. 

Instead this should pass in the `lightningID` of the destination node. I'll push a commit shortly which also allows creating a channel according to the destination`s `lightningID` rather than just the relative peer ID,